cleanCodeExercise.js

```diff
@@ -0,0 +1,70 @@
+// “Always code as if the guy who ends up maintaining your code will be a violent psychopath who knows where you live”
+// 
+// THIS IS A JAVASCRIPT-LIKE CODE IMPLEMENTING THE SCHEDULING OF BATHS AND VETERINARIANS ON THE PET STORE CASE.
+// 
+// PLEASE REVIEW THIS CODE TO GET IT AS CLEAN AS POSSIBLE. JUSTIFY YOUR CHANGES.
+// 
+
+
+var c;
+
+public class Costumer {
+
+	constructor(name, address, petName1, petName2, petName3, petName4, phoneNumber) {
+		this.name = height;
```

Comment:
O this.name no constructor está recebendo uma variavel com o nome diferente que inclusive nem está nos parametros do constructor.